Makefile

```diff
@@ -38,6 +43,25 @@ $(LOCAL_BINARY): $(SOURCES)
 test:
 	. ./scripts/shared_env && go test -v -timeout 30s -short -cover $(shell go list ./ecr-login/... | grep -v /vendor/)
 
+.PHONY: all-variants
+all-variants: linux-amd64 darwin-amd64 windows-amd64
```

Comment:
This is a great addition for this tool's build process :+1: 

With the general support for cross compilation in the official golang distribution as of 1.5, does it make sense to leave the `build` target instead of preferring to use `all-variants` in its place?

Reply:
I'd leave `build` as-is because I don't think everyone needs to build all variants all the time.